conda_smithy/lint_recipe.py

```diff
@@ -384,8 +384,27 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
                 )
             )
 
+    if recipe_dir:
+        forge_yaml_filename = (
+            glob(os.path.join(recipe_dir, "..", "conda-forge.yml"))
+            or glob(
+                os.path.join(recipe_dir, "conda-forge.yml"),
+            )
+            or glob(
+                os.path.join(recipe_dir, "..", "..", "conda-forge.yml"),
+            )
+        )
+        if forge_yaml_filename:
+            with open(forge_yaml_filename[0], "r") as fh:
```

Comment:
Shouldn't we check for `noarch_platforms` in all existing `conda-forge.yml`s, not just the first one we found? Or is it guaranteed that only one exists?

Reply:
There's only one used.